### PR TITLE
stm32_etzpc pm

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

File: core/include/kernel/pm.h [hunk 1]
```diff
@@ -133,6 +133,24 @@ static inline void register_pm_driver_cb(
...
+/*
+ * Register a core service callback for generic suspend/resume.
+ * Refer to struct pm_callback_handle for description of the callbacks
+ * API.
+ *
+ * @callback: Registered callback function
+ * @handle: Registered private handle argument for the callback
+ */
+static inline void register_pm_core_service_cb(
+		TEE_Result (*callback)(
```
Comment (jenswi-linaro) [inline core/include/kernel/pm.h]: This screams: give me a typedefed function pointer please!

Comment (etienne-lms) [reply core/include/kernel/pm.h]: addressed

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by etienne-lms** [comment]
fixup squashed and tag applied

**Pull request merged**